archetype_flask_connexion/database/db_utils.py

```diff
@@ -0,0 +1,65 @@
+"""Utility functions for MongoDB document insertion, updates and retrieval."""
+
+from typing import (Any, Mapping, Optional, Union)
+
+from bson.objectid import ObjectId
+from pymongo.collection import ReturnDocument
+from pymongo import collection as Collection
+
+
+def find_one_latest(collection: Collection) -> Optional[Mapping[Any, Any]]:
+    """Returns newest/latest object, stripped of the object id, or None if no
+    object exists: collection.
+    """
+    try:
+        return collection.find(
+            {},
+            {'_id': False}
+        ).sort([('_id', -1)]).limit(1).next()
+    except StopIteration:
+        return None
+
+
+def find_id_latest(collection: Collection) -> Optional[ObjectId]:
+    """Returns object id of newest/latest object, or None if no object exists.
+    """
+    try:
+        return collection.find().sort([('_id', -1)]).limit(1).next()['_id']
+    except StopIteration:
+        return None
+
+
+def update_task_state(
```

Comment:
This function should not be included, at least not for now, as it is specific to Celery